contracts/payloads/IGP105/PayloadIGP105.sol

```diff
@@ -0,0 +1,585 @@
+        action7();
+
+        // Action 8: Provide Credit to Team Multisig for Fluid DEX Lite
+        action8();
+    }
+
+    function verifyProposal() public view override {}
+
+    function _PROPOSAL_ID() internal view override returns (uint256) {
+        return PROPOSAL_ID;
+    }
+
+    /**
+     * |
+     * |     Proposal Payload Actions      |
+     * |__________________________________
+     */
+
+    // @notice Action 1: Clean up the default rate curves on unutilized assets
+    function action1() internal isActionSkippable(1) {
+        address[16] memory tokens = [
+            lBTC_ADDRESS, // lBTC
+            rsETH_ADDRESS, // rsETH
+            ezETH_ADDRESS, // ezETH
+            wstUSR_ADDRESS, // wstUSR
+            eBTC_ADDRESS, // eBTC
+            USD0_ADDRESS, // USD0
+            iUSD_ADDRESS, // iUSD
+            fxUSD_ADDRESS, // fxUSD
+            RLP_ADDRESS, // RLP
+            XAUT_ADDRESS, // XAUT
+            PAXG_ADDRESS, // PAXG
+            USR_ADDRESS, // USR
+            tBTC_ADDRESS, // tBTC
+            csUSDL_ADDRESS, // csUSDL
+            deUSD_ADDRESS // deUSD
+        ];
+
+        for (uint256 i = 0; i < tokens.length; i++) {
+            FluidLiquidityAdminStructs.RateDataV2Params[]
+                memory params_ = new FluidLiquidityAdminStructs.RateDataV2Params[](
+                    1
+                );
+            params_[0] = FluidLiquidityAdminStructs.RateDataV2Params({
+                token: tokens[i],
+                kink1: 50 * 1e2, // 50%
+                kink2: 80 * 1e2, // 80%
+                rateAtUtilizationZero: 0, // 0%
+                rateAtUtilizationKink1: 20 * 1e2, // 20%
+                rateAtUtilizationKink2: 40 * 1e2, // 40%
+                rateAtUtilizationMax: 100 * 1e2 // 100%
+            });
+            LIQUIDITY.updateRateDataV2s(params_);
+        }
+    }
+
+    // @notice Action 2: Update ETH interest rate curve
+    function action2() internal isActionSkippable(2) {
+        {
+            FluidLiquidityAdminStructs.RateDataV2Params[]
+                memory params_ = new FluidLiquidityAdminStructs.RateDataV2Params[](
+                    1
+                );
+
+            params_[0] = FluidLiquidityAdminStructs.RateDataV2Params({
+                token: ETH_ADDRESS, // ETH
+                kink1: 88 * 1e2, // 50%
+                kink2: 93 * 1e2, // 80%
+                rateAtUtilizationZero: 0, // 0%
+                rateAtUtilizationKink1: 2.75 * 1e2, // 2.75%
+                rateAtUtilizationKink2: 4 * 1e2, // 4%
+                rateAtUtilizationMax: 100 * 1e2 // 100%
+            });
+
+            LIQUIDITY.updateRateDataV2s(params_);
+        }
+    }
+
+    // @notice Action 3: Make Multisig 2 as deployer on all factories
+    function action3() internal isActionSkippable(3) {
+        // Set TEAM_MULTISIG_2 as deployer on all factories
+        IFluidVaultFactory(VAULT_FACTORY).setFactoryAuth(TEAM_MULTISIG_2, true);
```

Comment:
interface methods are different, can you check in code for each factory please? e.g.

https://github.com/Instadapp/fluid-contracts/blob/main/contracts/protocols/lending/lendingFactory/main.sol#L140